DEVELOPER.md

```diff
@@ -9,6 +9,8 @@ To generate a coverage report, there is a [coverage build script](https://github
 
 If you plan to contribute to Envoy, you may find it useful to install the Envoy [development support toolchain](https://github.com/envoyproxy/envoy/blob/main/support/README.md), which helps automate parts of the development process, particularly those involving code review.
 
+If you want to set up a local development environment with Visual Studio Code, please follow with [vscode setup](https://github.com/envoyproxy/envoy/blob/main/tools/vscode/README.md).
```

Comment:
"please follow with" sounds a bit wrong - how about changing it to just "please see"

Reply:
OK